### ISSUE TITLE
VNAV Bug during CRZ

### ISSUE THREAD

**Issue by oMrSmith** [mSparks43/747-400#777]
During CRZ
VNAV PTH can't decide on target mach speed
causing the speed bug to flicker
and the throttle to move erraticaly:

https://user-images.githubusercontent.com/78870770/163606206-d1fb0c3e-7f77-4f25-99b0-50f284ca59d8.mp4

**Comment by oMrSmith** [comment]
Seems like VNAV tries to bug for a speed above the speed limit...

**Comment by mSparks43** [comment]
this is "kinda" a known issue.
The cruise speed is set above the max safe speed, the protection system is setting it to a safe speed, but that works in kts, the VNAV system is working in mach and doesn't know the safe speed and it oscillates between set speed and safe speed.

The workaround is "dont do that".

**Comment by oMrSmith** [comment]
Wouldn't it be possible to have VNAV set a CRZ SPD that does not exceed the MAX safe speed?

**Comment by mSparks43** [comment]
> "dont do that" :D But the plane is doing it automatically... Wouldn't it be possible to have VNAV set a CRZ SPD that does not exceed the MAX safe speed?

Yes, but it needs either mach converting to kts or a better way of detecting that "it tried that already", neither of which were that simple compared to just not trying to set a mach speed faster than max safe speed.

Also dont actually know how it should behave - not found a reference for this case.

**Comment by oMrSmith** [comment]
@mSparks43 would it be possible to simply bypass the switch over to mach speed for now,
as it only causes trouble without giving any benefit?

**Comment by oMrSmith** [comment]
It seems like the flickering SPD Bug only happens with higher CI values...
I just flew my testroute form LGMK to LGAV with the CI set to 300
and the mach speed wasn't set above safe speed...
